public/sass/custom/kth-klaro.scss

```diff
@@ -58,96 +58,95 @@
   .cookie-notice {
     &:not(.cookie-modal-notice) {
       position: fixed;
-      right: auto;
-      top: auto;
-      left: 20px;
-      bottom: 20px;
-      background: none;
-      border-radius: 0;
-      z-index: 99999;
-
-      .cn-body {
-        font-size: 1rem;
-        padding: 1.5rem;
-        padding-top: 0.5rem;
-        position: absolute;
-        width: 554px;
-        height: fit-content;
-        max-width: 554px;
-        background-color: $white;
-        border-radius: 0.25rem;
-        box-shadow: 0 2px 10px 2px rgb(0 0 0 / 20%), 5px 5px 10px 0 rgb(0 0 0 / 19%);
-
-        .cn-ok {
-          flex-flow: column !important;
+      font-size: 1rem;
+      padding: 1.5rem;
+      margin: 10px;
```

Comment:
I wonder about padding in rem and margin in px. Is there any reason to not use the same unit for them?